cmd/restic/cmd_restore.go

```diff
@@ -125,11 +131,16 @@ func runRestore(opts RestoreOptions, gopts GlobalOptions, args []string) error {
 			Warnf("error for exclude pattern: %v", err)
 		}
 
+		matchedInsensitive, _, err := filter.InsensitiveList(opts.InsensitiveExclude, item)
```

Comment:
Unfortunately this is inefficient and does not work:
 * The function has a side effect of converting all strings in `opts.InsensitiveExclude` to lower case over and over again for every file that is to be restored
 * The `item` needs to be converted to lower case before passing it to the function

I suggest removing `InsensitiveList`, converting `opts.InsensitiveExclude` to lower case once, and then using `filter.List(opts.InsensitiveExclude, strings.ToLower(item)` instead. Thoughts?